controller/daq_site_config/mac_addrs/9a02571e8f03/device_config.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I would just call this controller/site_config (no daq_ prefix) -- the contents here are really about what goes on the controller machine, and it potentially has more uses outside of DAQ (e.g. used by cloud configuration tools).